### DISCUSSION TITLE
Summertime

### DISCUSSION THREAD

**Discussion by zaleslaw** [Kotlin/kotlindl#154] [Show and tell]
So, dear KotlinDL users and contributors, it was a hot May/June with 50 closed tickets and merged PRs from the community.
I'm going to say that I'll be on vacation from 26 June until 26 July with low Internet and very low ability to do reviews and participate in discussions.

All opened issues/PRs during this time will be answered after 26 July, excluding the easiest or urgent (related to top bugs).

All new PRs/issues could be done after reading [Contributing Guidelines](https://github.com/JetBrains/KotlinDL/blob/master/CONTRIBUTING.md).

Also, I hope that active contributors could help with answers to newcomers and with cross-review.
After my vacation, we will start preparing the 0.3 release, discuss the freeze dates, final scope, and make a Q&A session with active contributors and users in an online format.

Good summertime, dear community!

**Comment by zaleslaw** [comment]
@mkaze @avan1235 @therealansh @dosier @kokorins @d-lowl

**Reply by zaleslaw** [reply]
So, next week, I will have some time to answer issues and run TC builds or merge PRs but with low speed. Mention me in tickets if it's required to do something

**Reply by mkaze** [reply]
@zaleslaw Nice! Hope you enjoy your vacation time to the fullest! :smiley:

**Reply by dosier** [reply]
Hope u have a good time :)!

**Reply by zaleslaw** [reply]
Hi, community. KotlinDL is open for new PRs. Also, I've created a few new tickets, and I'm going to add more during this week; I hope you will have time to collaborate on something.
